assets/prometheus-k8s/rules.yaml

```diff
@@ -348,8 +348,8 @@ spec:
     - alert: NodeFilesystemSpaceFillingUp
       annotations:
         description: Filesystem on {{ $labels.device }} at {{ $labels.instance }}
-          has only {{ printf "%.2f" $value }}% available space left and is filling
-          up.
+          has only {{ printf "%.2f" $value | humanizePercentage }}% available space
```

Comment:
`has only {{  $value | humanizePercentage }} available`, the `%` character is automatically added by `humanizePercentage`.

Reply:
TIL. Thanks!